Tools-Override/tests.targets

```diff
@@ -214,6 +214,45 @@
       <PostExecutionTestCommandLines Include="copy /y testResults.xml %EXECUTION_DIR%\" />
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
```

Comment:
Are we sure this folder correctly maps between the WinRt documents and the Win32 document paths?